docker/Dockerfile.geth

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.21.2 AS build
+WORKDIR /volume
+
+RUN \
+# Install build dependencies
+    apk upgrade --no-cache \
+    && apk add --no-cache git make \
+# Clean up
+    && rm -rf /tmp/* \
+# Clone geth sources
+    && git clone https://github.com/ethereum/go-ethereum . \
+    && git checkout v1.14.5
+
+# Install go
+COPY --from=golang:1.22-alpine /usr/local/go/ /usr/local/go/
+
+# Add go to path
+ENV PATH="/usr/local/go/bin:${PATH}"
+
+# Build geth
+RUN make geth
+
+# Switch to clean container
+FROM alpine:3.21.2
+WORKDIR /volume
+
+# Copy built binary from build image
+COPY --from=build /volume/build/bin/geth /volume/geth
+
+# Set entrypoint to geth binary
+ENTRYPOINT ["/volume/geth",
+    # Ephemeral proof-of-authority network with a pre-funded developer account,
+    # with automatic mining when there are pending transactions.
+    "--dev",
+    "--datadir",
+    "./l1_datadir",
+    "--rpc.allow-unprotected-txs",
+    "--http",
+    "--http.addr",
+    "0.0.0.0",
+    "--http.port",
+    "58138",
+    "--http.corsdomain",
+    "*",
+    "--http.api",
+    "web3,debug,eth,txpool,net,engine"
+]
```

Comment:
We should expose the port via:
```
EXPOSE 58138
```

Reply:
And maybe respective ports for the op stack too? (e.g. 8560 for the proposer and so on)